OWNERS_ALIASES

```diff
@@ -44,3 +44,11 @@ aliases:
     - lance6716
     - tangenta
     - wjhuang2016
+  sig-approvers-table: # approvers for table packages.
```

Comment:
For the OLTP read write path, please add me.

Reply:
done